src/UnitySampleProject/Assets/InitTestScene15bc8b70-c00d-4f64-9cdd-a903c6d0d081.unity

```diff
@@ -0,0 +1,125 @@
+%YAML 1.1
```

Comment:
These are test scenes in `Assets` folder and they should be ignored

Reply:
lets remove them

Reply:
must of been left there, i went through like 100+ full tests lol will clean